tasking/manager.go

```diff
@@ -208,15 +202,11 @@ func (m *Manager) postpone(pending *model.Task, list []model.Task) (found bool)
 type Task struct {
 	// model.
 	*model.Task
-	// k8s client.
-	client client.Client
-	// addon
-	addon *crd.Addon
```

Comment:
Better for the Task to not have external references as state because they are Method (flow) dependent.